static-site/src/components/ListResources/ResourceGroup.tsx

```diff
@@ -15,9 +15,11 @@ interface ResourceGroupHeaderProps {
   isCollapsed: boolean
   resourcesToShowWhenCollapsed: number
   quickLinks: QuickLink[]
+  tooltipText: string
```

Comment:
There are quite a few tooltips within the resources UI, so I'd consider a prop name such as `defaultGroupLinksTooltip`.

Reply:
This is not the entire text and gets added to later. I would call it something like `groupTooltipPrefix`.